examples/websocket-node21-base/README.md

```diff
@@ -0,0 +1,72 @@
+
+# Node21 WebSocket Server
+
+This directory contains a Node21 WebSocket echo server implementation running on Unikraft.
+
+## Set Up
+
+To run this example, [install Unikraft's companion command-line toolchain `kraft`](https://unikraft.org/docs/cli), clone this repository, and `cd` into this directory.
+
+## Run and Use
+
+Use `kraft` to run the image and start a Unikraft instance locally:
+
+```bash
+kraft run --rm -p 8080:8080 --plat qemu --arch x86_64 -M 512M .
+```
+
+If the `--plat` argument is left out, it defaults to `qemu`.  
+If the `--arch` argument is left out, it defaults to your system's CPU architecture.
+
+Once running, it will open port `8080` and listen for WebSocket connections.
+
+To test it, you can use a WebSocket client like [`wscat`](https://github.com/websockets/wscat).
+
+Install `wscat` globally with `npm`:
+
+```bash
+npm install -g wscat
+```
+
+Then connect to the WebSocket server:
+
+```bash
+wscat --connect ws://localhost:8080
+```
+
+After connecting, you can enter messages, and the server will echo them back.
```

Comment:
Add here a snippet of such communication, so the user knows what to expect. Such as:

```text
Connected (press CTRL+C to quit)
< Connection received. Waiting for messages.
> hello
< hello
> bye
< bye
```